RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# How to release edtr-io
+
+## Updating all packages:
+
+1. `yarn build` - run it locally to check if the build will pass
+2. `yarn regenerate-api` - run it locally to check if the generated API documentation is up to date
+3. `yarn prepare-release` - updates the Lerna version (which will be read by the GitHub deploy workflow (`/.github/workflows/deploy.yml`) once the release commit is pushed)
+4. Add a changelog entry in `/scripts/changelog.ts` and then run `yarn changelog` to generate a new changelog entry in `CHANGELOG.md`
+5. `git commit -m "release X.X.X"` - commit the changes with the correct release tag
```

Comment:
Unclear, if on branch main or other branch. Maybe add a short note at the start, that we perform all these steps on branch main?